.travis.yml

```diff
@@ -2,12 +2,15 @@ language: python
 cache:
   directories:
     - $HOME/.cargo
-    - cargo
     - autopush_rs/target
     - $HOME/.cache/pip
 sudo: required
 dist: trusty
 
+env:
+  global:
+    - WITH_RUST=release
```

Comment:
I was hoping to switch travis to rustc debug mode but that enables slog's verbose logging that floods travis w/ 4mb of logs, causing tests to fail (we can revisit this later)